File: Lez03_06/Lez03_06/classes/Animale.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lez03_06.classes
{
    internal abstract class Animale
    {
        public bool HasPelo { get; set; }
        public int NumZampe { get; set; }
        public bool Vola { get; set; }

        public abstract void VersoEmesso();
    }
}

File: Lez_TS/lez12_poly/Animale.ts
export default abstract class Animale{
    private hasPelo: boolean = false;

    constructor(hasPelo: boolean){
        this.hasPelo = hasPelo;
    }

    abstract versoEmesso(): void;

    dormi(): void{
        console.log("zzzzZZZZZZ")
    }
}